src/components/data/DataNavigation.js

```diff
@@ -0,0 +1,460 @@
+    const router = useRouter();
+    const classes = useStyles();
+    const relativePath = getRelativePath(path, userHomePath, userTrashPath);
+
+    const handleClick = (event, relativePath, crumb) => {
+        event.preventDefault();
+        const relativePathItems = getPathItems(relativePath);
+        const index = relativePathItems.indexOf(crumb);
+        router.push(pathToRoute(root, relativePath, index));
+    };
+
+    return (
+        <Breadcrumbs
+            maxItems={2}
+            style={{ paddingTop: 8 }}
+            aria-label="breadcrumb"
+            id={build(baseId, ids.BREADCRUMBS)}
+            aria-controls={formatMessage(
+                intl,
+                "dataNavigationBreadcrumbsControl"
+            )}
+        >
+            {getPathItems(relativePath).map((crumb) => (
+                <>
+                    <ArrowRightIcon />
+                    <FolderIcon color="primary" />
+                    <Tooltip title={crumb}>
+                        <Link
+                            aria-controls={formatMessage(
+                                intl,
+                                "folderPathAriaControl"
+                            )}
+                            id={build(
+                                baseId,
+                                ids.BREADCRUMBS,
+                                ids.PATH_ITEM,
+                                crumb
+                            )}
+                            key={crumb}
+                            color="inherit"
+                            href="#"
+                            onClick={(event) =>
+                                handleClick(event, relativePath, crumb)
+                            }
+                            className={classes.breadCrumbLink}
+                        >
+                            {crumb}
+                        </Link>
+                    </Tooltip>
+                </>
+            ))}
+        </Breadcrumbs>
+    );
+}
+
+function DataNavigation(props) {
+    const {
+        path,
+        error,
+        baseId,
+        dataRoots,
+        userHomePath,
+        userTrashPath,
+        intl,
+    } = props;
+    const router = useRouter();
+    const classes = useStyles();
+    const [anchorEl, setAnchorEl] = useState(null);
+    const [selectedIndex, setSelectedIndex] = useState(0);
+    const dataNavId = build(baseId, ids.DATA_NAVIGATION);
+
+    const handleClickListItem = (event) => {
+        setAnchorEl(event.currentTarget);
+    };
+
+    const handleMenuItemClick = (event, index) => {
+        setSelectedIndex(index);
+        setAnchorEl(null);
+        router.push(
+            "/" + NavigationConstants.DATA + `?path=${dataRoots[index].path}`
+        );
+    };
+
+    const handleClose = () => {
+        setAnchorEl(null);
+    };
+
+    if (dataRoots.length === 0) {
+        return null;
+    }
+    return (
+        dataRoots.length > 0 && (
```

Comment:
Since there's already a check above that returns `null`, this one seems unnecessary.